install_config/web_console_customization.adoc

```diff
@@ -991,16 +972,54 @@ assetConfig:
 +
 ----
 ifdef::openshift-origin[]
-# systemctl restart origin-master
+# systemctl restart origin-master-api origin-master-controllers
 endif::[]
 ifdef::openshift-enterprise[]
-# systemctl restart atomic-openshift-master
+# systemctl restart atomic-openshift-master-api atomic-openshift-master-controllers
 endif::[]
 ----
 
 xref:../install_config/router/default_haproxy_router.adoc#using-wildcard-routes[Learn
 how to configure HAProxy routers to allow wildcard routes].
 
+[[web-console-enable-tech-preview-feature]]
+== Enabling Features in Technology Preview
+
+Sometimes features are available in Technology Preview. By default, these
+features are disabled and hidden in the web console.
+
+Currently, there are no web console features in Technology Preview.
+
+To enable a Technology Preview feature:
+
+. Save this script to a file (for example, *_tech-preview.js_*):
++
+[source, javascript]
+----
+window.OPENSHIFT_CONSTANTS.ENABLE_TECH_PREVIEW_FEATURE.<feature_name> = true;
+----
```

Comment:
The above confused me a little. Is there reader creating the tech-preview.js file, then adding above line to it? Is that all? And how do they know what the feature name is for the feature?

Reply:
@bfallonf This PR was specific to changing the master service names. I can open a follow-up to persue these matters.